ethereum_history_api/oracles/src/fixtures/config.ts

```diff
@@ -65,6 +66,10 @@ export const FIXTURES: Fixtures = {
       },
       access_list: {
         blockNumber: 19_439_366n
+      },
+      transfer: {
+        blockNumber: 19_539_214n,
+        transactionHash: TRANSFER_TX_HASH
```

Comment:
We already have CHAIN_LINK_TRANSFER_TX_HASH. How is this one different? If we can't reuse that one - let's express why in the variable name